core/baseline-core-v1.0-psd01.md

```diff
@@ -2792,105 +2792,156 @@ As has been done throughout this document, there are BPI layer-specific security
+
 #### **[R284]**	
 BPI Storage MUST support policy management.
 
+[[R284]](#r284) Testability: Common BPI storage options such as [PostgreSQL](https://www.postgresql.org/) with their comprehensive test suite of all functionality support Policy Management such as [row based policy management](https://www.postgresql.org/docs/current/ddl-rowsecurity.html). Therefore, the requirement is testable.
+
 #### **[R285]**	
 BPI Storage MUST support Single-Sign-On (SSO).
 
 *See [[SSO](#sso)] also for the recommendations of the National Institute of Standards and Technology (NIST Guide to Secure Web Services).*
 
+[[R285]](#r285) Testability: Common BPI storage options such as [PostgreSQL](https://www.postgresql.org/) with their comprehensive test suite of all functionality support [SSO](https://www.postgresql.org/docs/10/auth-methods.html). Therefore, the requirement is testable.
+
 #### **[R286]**	
 BPI Storage MUST support multi-factor authentication (MFA)
 
 *See the link here for the NIST definition adopted in this document [MFA](https://csrc.nist.gov/glossary/term/mfa).*
 
+[[R286]](#r286) Testability: Common BPI storage options such as [PostgreSQL](https://www.postgresql.org/) with their comprehensive test suite of all functionality support MFA by allowing up to three authentication factors in a user authentication policy, see [user creation](https://dev.mysql.com/doc/refman/8.0/en/create-user.html). Therefore, the requirement is testable.
+
 #### **[R287]**	
 BPI Storage MUST support hardware security modules (HSM). 
 
 *This document adopts the [NIST definition](https://csrc.nist.gov/glossary/term/hardware_security_module_hsm) and for further information, refer to [[HSM](#hsm)].*
 
+[[R287]](#r287) Testability: Common BPI storage options such as [PostgreSQL](https://www.postgresql.org/) with their comprehensive test suite of all functionality support [cryptographic key based authentication frameworks such as Oauth-2.0](https://www.postgresql.org/docs/10/auth-methods.html). Since the private keys associated with the public keys used in cryptographic key based authentication frameworks can be managed in HSM-based services such as [AWS KMS](https://aws.amazon.com/kms/), the requirement is testable.
+
 ## 7.2. BPI Storage Privacy
 
 As has been done throughout this document, there are BPI layer-specific privacy requirements for layers and/or components which are listed below. It is assumed that the BPI Storage capabilities are required to fulfill all BPI privacy requirements in section [2 Design and Architecture](#2-design-and-architecture).
 
 #### **[R288]**	
 Personal Identifiable Information (PII) MUST NOT be stored in a BPI.
 
+[[R288]](#r288) Testability: Since BPI Storage supports testable policy management (see [[R284]](#r284)), and since preventing PII storage can be cast into a policy, the requirement is testable because of testability of [[R284]](#r284).
```

Comment:
I don't think a policy that says "don't put PII here" solves the issue. On the other hand, if the information stored can be assessed (like a security review) then I think it is testable whether PII is stored there or not.

```suggestion
[[R288]](#r288) Testability: Since BPI Storage can be reviewed to determine whether PII has been stored, the requirement is testable because).
```